lib/openfeature/rollout/provider.rb

```diff
@@ -64,23 +64,23 @@ def fetch_boolean_value(flag_key:, default_value:, evaluation_context: nil)
         )
       end
 
-      def fetch_number_value(flag_key:, default_value:, evaluation_context: nil)
+      def fetch_number_value(flag_key:, default_value: nil, evaluation_context: nil)
```

Comment:
I think these are fine to leave as nil, since they will always error in every situation.